packaging/irods_client_rest_cpp.json.template

```diff
@@ -1,52 +1,62 @@
 {
-    "irods_rest_cpp_access_server" : {
-        "port" : 8080,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_admin_server" : {
-        "port" : 8087,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_auth_server" : {
-        "port" : 8081,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_get_configuration_server" : {
-        "port" : 8088,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_put_configuration_server" : {
-        "port" : 8089,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_list_server" : {
-        "port" : 8082,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_query_server" : {
-        "port" : 8083,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_stream_get_server" : {
-        "port" : 8084,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_stream_put_server" : {
-        "port" : 8085,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
-    },
-    "irods_rest_cpp_zone_report_server" : {
-        "port" : 8086,
-        "threads" : 4,
-        "maximum_idle_timeout_in_seconds" : 10
+    "irods_rest_cpp_access_server": {
+        "port": 8080,
+        "threads": 4,
+        "maximum_idle_timeout_in_seconds": 10,
+        "log_level": "info"
```

Comment:
@trel Here is the default log level. Is `info` what we want?

Reply:
Without additional context/usecases/reasoning... I think so - let's start there.